### DISCUSSION TITLE
Fractal noise usage

### DISCUSSION THREAD

**Discussion by antoine-perez** [bnpr/Malt#191] [Q&A]
How is the fractal noise node meant to be used? What should I plug into the coord (vec4) socket? I'd like to generate a world-space 4D noise to displace vertices.

**Comment by pragma37** [comment]
Hello Antoine.
The coord input should be the coordinates used for the noise.
Typically the first 3 values will be the surface position, and the fourth value can be used either as a seed or as a time parameter.

Here's an example:
![imagen](https://user-images.githubusercontent.com/36610999/145610760-6323faea-35b6-4300-a862-9026e6e0896a.png)

[fractal_noise_displace.zip](https://github.com/bnpr/Malt/files/7694218/fractal_noise_displace.zip)
